Vagrantfile

```diff
@@ -81,7 +81,7 @@ Vagrant.configure("2") do |config|
      ./gtminstall --utf8 default
      gtmver=$(ls -1 /usr/lib/fis-gtm)
      echo "source /usr/lib/fis-gtm/$gtmver/gtmprofile" >> /home/ubuntu/.profile
-     ln -s $gtm_dist/libgtmshr.so /usr/local/lib
+     ln -s $gtm_dist/libgtmshr.so /vagrant/lib
```

Comment:
Yes, the rest of the file can be moved to /vagrant. This line is very specific. *.so files are like dlls on windows, etc (it is also a binary file). The operating systems looks for dlls in very specific places and on ubuntu /usr/local/lib is one of the places it searches.

These are also operating system dependent (windows vs linux vs mac, etc.) and platform dependent (x32 vs x64 vs mips vs arm, etc.) so sharing them with the host doesn't gain anything.